roles/icinga2-primary/templates/opsgenie/config.json

```diff
@@ -0,0 +1,63 @@
+{
+  "apiKey": "{{ icinga2_opsgenie_key }}",
+  "baseUrl": "https://api.eu.opsgenie.com",
+  "logLevel": "DEBUG",
+  "globalArgs": [],
+  "globalFlags": {
+    "graphite_url": "http://localhost:5003",
+    "api_url": "https://localhost:5665",
+    "user": "icingaadmin",
+    "password": "icingaadmin",
```

Comment:
Is the password of the icingaadmin user really icingaadmin? If yes we should change it, shouldn't we?

Reply:
We do not have a user called icingaadmin. this would only be needed if we enabled the connection from opsgenie to our icinga (e.g. to report acknowledgements etc) but i would like to keep this disabled.